### PR TITLE
update to latest version

### PR DESCRIPTION
**Pull request by kayarre**
Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

 <code>@<space/>conda-forge-admin, please rerender</code>

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/t/tifffile/tifffile-{{ version }}.tar.gz
-  sha256: 1fbb2cfd57fd8e42e417bc29001a17f319701f1be00e0b8a0004a52da93f1b08
+  sha256: 82c5c72de4dc19cc7011e4e8c45492e17121bd02cfa98c015ddd2a83e36f09bc
 
 build:
-    number: 1001
+    number: 1000
```
Comment (ocefpaf) [inline recipe/meta.yaml]: Please set it to `0`.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,15 +1,15 @@
-{% set version = "0.15.1" %}
+{% set version = "2019.7.26" %}
```
Comment (ocefpaf) [inline recipe/meta.yaml]: Is this older or newer than 0.15.1 :smile:

Comment (kayarre) [reply recipe/meta.yaml]: @ocefpaf it appears the versioning scheme changed between `0.15.1` and `2018.10.18`

File: recipe/meta.yaml (hunk 1 above)
Comment (kayarre) [reply recipe/meta.yaml]: updated

**Pull request merged**